sdk/communication/communication-sms/src/smsClient.ts

```diff
@@ -135,20 +165,36 @@ export class SmsClient {
    * @param sendRequest - Provides the sender's and recipient's phone numbers, and the contents of the message
    * @param options - Additional request options
    */
-  public async send(sendRequest: SendRequest, options: SendOptions = {}): Promise<RestResponse> {
-    const { operationOptions, restOptions } = extractOperationOptions(options);
-    const { span, updatedOptions } = createSpan("SmsClient-send", operationOptions);
+  public async send(
+    _sendRequest: SmsSendRequest,
```

Comment:
you can remove the '_' from front since that was added to avoid the unused variable errors